pong.pde

```diff
@@ -16,6 +16,51 @@ var dist_wall = 8;
 float highscore = 0;
 float score = 0;
 
+
+
+void helperFunction(){
```

Comment:
This code looks good to go and should work when deployed. Was not able to create a separate environment to test this branch before merging it with the main branch.

Good to go - tentatively.